src/components/EditCoursePage/AdditionalMetadataFields.jsx

```diff
@@ -41,6 +41,19 @@ const AdditionalMetadataFields = (props) => {
     <div className="collapsible-card pgn_collapsible mt-4">
       <div className="collapsible-body">
         <FieldLabel text="Additional Metadata Details" className="h3 font-weight-normal mb-3" />
+        <div>
+          <Field
+            name="additional_metadata.display_on_org_page"
+            type="checkbox"
+            label="additional_metadata.display_on_org_page"
+            disabled={disabled}
+            component="input"
+            required={isRequiredField('display_on_org_page')}
+            className="float-left mt-1"
+          />
+          <FieldLabel text="Display on Organization Page" className="font-weight-normal ml-2 float-left" />
+          <br />
+        </div>
```

Comment:
I believe margin or some spacing would be good here

<img width="1117" alt="Screenshot 2023-07-24 at 5 31 22 PM" src="https://github.com/openedx/frontend-app-publisher/assets/40599381/eb97e0a5-98bb-465f-bab2-770358c18cf7">

Reply:
Sure.